packages/appeals-service-api/src/services/back-office-v2/formatters/has/appeal.js

```diff
@@ -22,57 +23,52 @@ exports.formatter = async (appellantSubmission) => {
 	}
 
-	// const address = appellantSubmission.SubmissionAddress?.find(
-	// 	(address) => address.fieldName === 'siteAddress'
-	// );
+	const address = appellantSubmission.SubmissionAddress?.find(
+		(address) => address.fieldName === 'siteAddress'
+	);
 
 	return [
 		{
-			appeal: {
-				LPACode: lpa.getLpaCode(), // use Q9999 format, appeal uses E69999999
-				LPAName: lpa.getName(),
-				appealType: 'Householder (HAS) Appeal',
-				isListedBuilding: false,
-				decision: appellantSubmission.applicationDecision,
-				// originalCaseDecisionDate: appellantSubmission.applicationDecisionDate.toISOString(),
-				costsAppliedForIndicator: appellantSubmission.costApplication,
-				LPAApplicationReference: appellantSubmission.applicationReference,
-				appellant: {
-					firstName: appellantSubmission.isAppellant
-						? appellantSubmission.contactFirstName
-						: appellantSubmission.appellantFirstName,
-					lastName: appellantSubmission.isAppellant
-						? appellantSubmission.contactLastName
-						: appellantSubmission.appellantLastName,
-					emailAddress: appellantSubmission.appellantEmailAddress
-				},
-				agent: !appellantSubmission.isAppellant
-					? {
-							firstName: appellantSubmission.contactFirstName,
-							lastName: appellantSubmission.contactLastName,
-							emailAddress: appellantSubmission.appellantEmailAddress
-					  }
-					: undefined,
-				// siteAddressLine1: address.addressLine1,
-				// siteAddressLine2: address.addressLine2,
-				// siteAddressTown: address.town,
-				// siteAddressCounty: address.county,
-				// siteAddressPostcode: address.postcode,
-				isSiteFullyOwned: appellantSubmission.ownsAllLand,
-				hasToldOwners: !appellantSubmission.ownsAllLand
-					? appellantSubmission.informedOwners
-					: undefined,
-				isSiteVisible: appellantSubmission.appellantSiteAccess,
-				inspectorAccessDetails: appellantSubmission.appellantSiteAccess
-					? appellantSubmission.appellantSiteAccess_appellantSiteAccessDetails
-					: undefined,
-				doesSiteHaveHealthAndSafetyIssues: appellantSubmission.appellantSiteSafety,
-				healthAndSafetyIssuesDetails: appellantSubmission.appellantSiteSafety
-					? appellantSubmission.appellantsiteSafety_appellantSiteAccessDetails
-					: undefined
+			casedata: {
+				caseType: 'D',
+				caseProcedure: 'written', // We keep this on Appeal cases, will one exist yet when this code is triggered?
+				lpaCode: lpa.getLpaCode(),
+				caseSubmittedDate: new Date().getTime().toString(),
+				enforcementNotice: null, // I can't find anywhere where we collect this
+				applicationReference: appellantSubmission.applicationReference ?? '',
+				applicationDate: new Date().getTime().toString(), // Now presumably?
+				applicationDecision: appellantSubmission.applicationDecision ?? 'not-received',
+				applicationDecisionDate: appellantSubmission.applicationDecisionDate ?? null,
+				caseSubmissionDueDate: new Date().getTime().toString(), // Now presumably?,
```

Comment:
this is a calculation based on the decision date + appeal type + legal rules, it is not held in db.
I've been planning to run calculation here before we send over.
See: /business-rules/rules/appeal/deadline-date.js

Reply:
perfect, thanks! used